Bixal/move.mil
# Feature 157853900 remove l r pro tip alignment

## Checklist

I have…

      against my changes

## Summary of Changes

This pull request…

- removes the styles and template logic for R|L aligned pro-tip paragraphs

## Testing

To verify the changes proposed in this pull request…

1. pull changes locally
2. from theme root, npm run build
3. make cr
4. navigate to /moving-guide/oconus
5. from the editor view update all pro-tip to bottom aligned

***# **note**: we have to update all of the R|L aligned pro tip paragraphs before removing and updating config for this field. Since this has to be done upstream on the DB, config will be exported in a separate PR.

## Screenshots

![screencapture-move-mil-localhost-8000-moving-guide-oconus-2018-06-01-11_24_20](https://user-images.githubusercontent.com/37077057/40849229-570e2794-658f-11e8-8275-251411734cbf.png)
